rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.aggregation/41_date_range.yml

```diff
@@ -0,0 +1,178 @@
+setup:
+  - do:
+      indices.create:
+          index: test
+          body:
+            settings:
+              number_of_replicas: 0
+            mappings:
+              properties:
+                date:
+                  type: date
+                  format: epoch_second
+
+  - do:
+      cluster.health:
+        wait_for_status: green
+
+---
+"Date range":
+  - do:
+      index:
+        index: test
+        id: 1
+        body: { "date" : 1000 }
+
+  - do:
+      index:
+        index: test
+        id: 2
+        body: { "date" : 2000 }
+
+  - do:
+      index:
+        index: test
+        id: 3
+        body: { "date" : 3000 }
+
+  - do:
+      indices.refresh: {}
+
+  - do:
+      search:
+        rest_total_hits_as_int: true
```

Comment:
At some point we're going to deprecate this thing so I'd try to avoid adding it to master. It is sort of required if you are running these against 6.x, but maybe only add on backport? I dunno.

Reply:
Ugh, I hadn't even thought of that.  We use that all over the place in these tests